src/reviews/views.py

```diff
@@ -55,6 +55,13 @@ def get_ordering(self):
         order_key = self.order_keys.get(params.get('order', '').lower())
         return order_key or '?'
 
+    def get_category(self):
+        params = self.request.GET
+        category_key = params.get('category', '').upper()
+        if category_key:
+            return category_key
+        return ''
```

Comment:
Looks like this is enough?

```suggestion
        return category_key
```

Reply:
Or we could

```python
category_key = self.request.GET.get('category')
if category_key:
    category_key.upper()
return ''
```